tests/aske/test_GrFN.py

```diff
@@ -4,93 +4,149 @@
-    assert res == np.float32(0.00012496980836348878)
+    assert res[0] == np.float32(0.00012496980836348878)
 
 
 def test_crop_yield_creation(crop_yield_grfn):
-    A = crop_yield_grfn.to_agraph()
     assert isinstance(crop_yield_grfn, GroundedFunctionNetwork)
-
-
-@pytest.mark.skip
-def test_petasce_torch_execution():
-    lambdas = importlib.__import__("PETASCE_simple_torch_lambdas")
-    pgm = json.load(open(data_dir + "PETASCE_simple_torch.json", "r"))
-    G = GroundedFunctionNetwork.from_dict(pgm, lambdas)
-
-    N = 100
-    samples = {
-        "petasce::doy_0": np.random.randint(1, 100, N),
-        "petasce::meevp_0": np.where(np.random.rand(N) >= 0.5, 'A', 'W'),
-        "petasce::msalb_0": np.random.uniform(0, 1, N),
-        "petasce::srad_0": np.random.uniform(1, 30, N),
-        "petasce::tmax_0": np.random.uniform(-30, 60, N),
-        "petasce::tmin_0": np.random.uniform(-30, 60, N),
-        "petasce::xhlai_0": np.random.uniform(0, 20, N),
-        "petasce::tdew_0": np.random.uniform(-30, 60, N),
-        "petasce::windht_0": np.random.uniform(0, 10, N),
-        "petasce::windrun_0": np.random.uniform(0, 900, N),
-        "petasce::xlat_0": np.random.uniform(0, 90, N),
-        "petasce::xelev_0": np.random.uniform(0, 6000, N),
-        "petasce::canht_0": np.random.uniform(0.001, 3, N),
-    }
-
-    values = {
-        k: torch.tensor(v, dtype=torch.double) if v.dtype != "<U1" else v
-        for k, v in samples.items()
-    }
-
-    res = G.run(values, torch_size=N)
-    assert res.size()[0] == N
+    G = crop_yield_grfn.to_agraph()
+    G.draw('crop_yield--GrFN.pdf', prog='dot')
+    CAG = crop_yield_grfn.to_CAG_agraph()
+    CAG.draw('crop_yield--CAG.pdf', prog='dot')
+
+
+def test_sir_simple_creation(sir_simple_grfn):
+    assert isinstance(sir_simple_grfn, GroundedFunctionNetwork)
+    G = sir_simple_grfn.to_agraph()
+    G.draw('SIR-simple--GrFN.pdf', prog='dot')
+    CAG = sir_simple_grfn.to_CAG_agraph()
+    CAG.draw('SIR-simple--CAG.pdf', prog='dot')
+    lambdas = importlib.__import__(f"SIR-simple_lambdas")
+    (D, I, S, F) = to_wiring_diagram(sir_simple_grfn, lambdas)
+    assert len(D) == 3
+    assert len(I) == 3
+    assert len(S) == 9
+    assert len(F) == 5
+
+
+def test_sir_gillespie_inline_creation(sir_gillespie_inline_grfn):
+    assert isinstance(sir_gillespie_inline_grfn, GroundedFunctionNetwork)
+    G = sir_gillespie_inline_grfn.to_agraph()
+    G.draw('SIR-Gillespie_inline--GrFN.pdf', prog='dot')
+    CAG = sir_gillespie_inline_grfn.to_CAG_agraph()
+    CAG.draw('SIR-Gillespie_inline--CAG.pdf', prog='dot')
+
+
+def test_sir_gillespie_ms_creation(sir_gillespie_ms_grfn):
+    assert isinstance(sir_gillespie_ms_grfn, GroundedFunctionNetwork)
+    G = sir_gillespie_ms_grfn.to_agraph()
+    G.draw('SIR-Gillespie_ms--GrFN.pdf', prog='dot')
+    CAG = sir_gillespie_ms_grfn.to_CAG_agraph()
+    CAG.draw('SIR-Gillespie_ms--CAG.pdf', prog='dot')
+
+
+# def test_petasce_torch_execution():
```

Comment:
Can we delete this commented-out code?

Reply:
My mistake, I still want this test in the future so I'll use the pytest skip fixture as opposed to commenting out the code.